lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -599,6 +593,13 @@ def main():
             os.remove(tmpsrc)
             module.fail_json(msg="Destination %s is not writable" % (os.path.dirname(dest)), **result)
 
+    if module.check_mode:
+        if os.path.exists(tmpsrc):
+            os.remove(tmpsrc)
+        result['changed'] = ('checksum_dest' not in result or
```

Comment:
You'd better add a test for `'checksum_dest' not in result`.

Reply:
I think the existing check_mode tests already cover that case, or have I missed something?

Reply:
You are right.

the following test case covers your code.
 
> name: test HTTP HEAD request for file in check mode